Code/Editor/CryEdit.cpp

```diff
@@ -162,6 +163,10 @@ static const char O3DEApplicationName[] = "O3DEApplication";
 
 static AZ::EnvironmentVariable<bool> inEditorBatchMode = nullptr;
 
+namespace Platform {
+    bool OpenUri(const QUrl& uri);
+}
+
```

Comment:
coding standard: curly brace should be on a line by itself: https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#positioning-of-curly-braces
```suggestion
namespace Platform
{
    bool OpenUri(const QUrl& uri);
}

```

Reply:
Thanks will fix.